src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -3451,6 +3453,73 @@ void ScreenBufferTests::ReverseLineFeedInMargins()
     }
 }
 
+void ScreenBufferTests::DeleteLines256Colors()
+{
+    // This test is largely taken from repro code from
+    // https://github.com/microsoft/terminal/issues/832#issuecomment-507447272
+    Log::Comment(
+        L"Sets the attributes to a 256/RGB color, then scrolls some lines with"
+        L" DL. Verifies the rows are cleared with the attributes we'd expect.");
+
+    auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
+    auto& si = gci.GetActiveOutputBuffer();
+    auto& tbi = si.GetTextBuffer();
+    auto& stateMachine = si.GetStateMachine();
+    auto& cursor = si.GetTextBuffer().GetCursor();
+
+    TextAttribute expectedAttr{ si.GetAttributes() };
+    expectedAttr.SetBackground(gci.GetColorTableEntry(2));
+
+    // Set some scrolling margins
+    stateMachine.ProcessString(L"\x1b[1;3r");
+
+    // Set the BG color to the table index 2, as a 256-color sequence
+    stateMachine.ProcessString(L"\x1b[48;5;2m");

```

Comment:
actually, does 256-color `2` match 16-color `2`? I'd be concerned that this isn't testing something above the legacy attributes if so.